README.md

```diff
@@ -170,7 +168,7 @@ Using this flow is less secure since the token is long-lived. You can provide th
 
 ## Reporting issues
 
-If you encounter any issues or have suggestions for improvements, please open an issue in the repository.
+If you encounter any issues or have suggestions for improvements, please open an issue in the repository or create a ticket in the [STACKIT Help Center](https://support.stackit.cloud/).

```

Comment:
Is the STACKIT help center publicly available?

Reply:
Its publicly available for STACKIT customers

Reply:
But the answer for your question is: yes